sympy/printing/pretty/pretty.py

```diff
@@ -531,7 +531,7 @@ def adjust(s, wid=None, how='<^>'):
         for lim in expr.limits:
             if len(lim) == 3:
                 prettyUpper = self._print(lim[2])
-                prettyLower = self._print(Equality(lim[0], lim[1]))
+                prettyLower = self._print(str(lim[0]) + " = " + str(lim[1]))
```

Comment:
One of the errors in the travis report seems to be that a dummy variable is printed as "_k" when it expects it to be printed as "k" - this is because str(Dummy('k')) = "_k". Changing `str(lim[0])` to `lim[0].name` should do the trick.